src/docfx/build/schema/BuildSchemaDocument.cs

```diff
@@ -38,10 +39,11 @@ internal static class BuildSchemaDocument
             var (id, versionIndependentId) = file.Docset.Redirections.TryGetDocumentId(file, out var docId) ? docId : file.Id;
 
             // TODO: add check before to avoid case failure
-            var (repoErrors, author, contributors, updatedAt) = contribution.GetContributorInfo(
+            var (repoErrors, author, contributors, updatedAt) = await contribution.GetContributorInfo(
                 file,
                 metadata.Value<string>("author"),
                 metadata.Value<DateTime?>("update_date"));
+            errors.AddRange(repoErrors);
```

Comment:
format?

Reply:
what format issue do you mean?

---
In reply to: [207149765](https://github.com/dotnet/docfx/pull/3144#discussion_r207149765) [](ancestors = 207149765)